README.md

```diff
@@ -57,5 +57,5 @@ In order to be able to launch the bot server locally you have to compile sources
 
 + `IS_LONGPOLL=true` for using long polling instead of web hooking (common usage case is local launch and debugging)
 
-<img src="info/logo.jpg" width=100%>
+[![Pull Party Bot](info/logo.jpg)](https://t.me/PullPartyBot)
```

Comment:
We have to revert it back to html, it seems bad on the big screens.

Need to revert to `width=100%` or set `align=center`